helpers/mathjax.js

```diff
@@ -1,5 +1,36 @@
 let mathJaxLoaded;
 
+export class HtmlBlockMathRenderer {
+
+	get contextAttribute() {
+		return 'data-mathjax-context';
```

Comment:
Could there be a use-case where a renderer depends on more than one attribute?

Reply:
Hmm, that's a good question. I envisioned these (broadly speaking) as single context attributes specific to each renderer, similar to how the HTML editor context object currently works. I'm unsure if there's value in supporting the multiple-attribute case, given that these can be arbitrary objects as it is. 🤔

Reply:
That said, it would be trivial to support this now, so perhaps it would be better to just do so.